frontend/src/citizen-frontend/decisions/api.ts

```diff
@@ -37,9 +37,15 @@ export async function acceptDecision(
   decisionId: UUID,
   requestedStartDate: string
 ): Promise<Result<void>> {
+  let date
+  try {
+    date = LocalDate.parseFi(requestedStartDate)
+  } catch (e) {
+    return Failure.fromError(e)
+  }
```

Comment:
wouldn't it make more sense to handle parsing and validation errors at the component. this api call should take LocalDate instead of string.